src/Backend/Form/Type/CheckboxEnableField.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Backend\Form\Type;
+
+use Backend\Form\DataTransferObject\EnabledFieldDataTransferObject;
+use Common\Form\SwitchType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CheckboxEnableField extends AbstractType
```

Comment:
your DTO is called EnabledField and your form type is called CheckboxEnableField
Don't like this naming inconsistency.
Plus all form types need to have the suffix Type

Reply:
fixed 42f2d5c18ecef0a634f028a335a02722b02c9528